scripts/training/prepare_data.py

```
@@ -0,0 +1,213 @@
+        items.append(d)
+    return items
+
+
+def split(pairs, ratios=(0.8, 0.1, 0.1), seed=42):
+    rng = random.Random(seed)
+    # 按 scenario 分组保证隔离
+    by_scn = {}
+    for p in pairs:
+        by_scn.setdefault(p["scenario"], []).append(p)
+    train, val, test = [], [], []
+    for scn, lst in by_scn.items():
```

Ruff (0.16.3)

[warning] Loop control variable `scn` not used within loop body

Rename unused `scn` to `_scn`

(B007)